lib/portal.js

```diff
@@ -180,7 +180,16 @@ Portal.propTypes = {
   openByClickOn: React.PropTypes.element,
   closeOnEsc: React.PropTypes.bool,
   closeOnOutsideClick: React.PropTypes.bool,
-  isOpened: React.PropTypes.bool,
+  isOpen: React.PropTypes.bool,
+  isOpened: (props, propName, componentName) => {
+    if (typeof props[propName] !== 'undefined') {
+      return new Error(
+          `Deprecated prop \`${propName}\` supplied to \`${componentName}\`.
+          Please use \`isOpen\` in the future.`
```

Comment:
"In the future" might be misleading here - this is an error, you _have_ to use `isOpen` for the portal to work, right?